2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -702,5 +702,70 @@ A causa dei rallentamenti incontrati, alcuni ruoli hanno visto un monte ore effe
 
 - Verificatore: i numerosi merge conflicts hanno portato a notevoli rallentamenti in fase di review.
 
+=== Burndown-chart
+[da inserire]
+
+
+== Quinto Sprint dal 04-12-2023 al 11-12-2023
+
+=== Obiettivi raggiunti
+- miglioramento e aggiornamento documento Analisi dei Requisiti;
+- realizzazione mini PoC relativo alla creazione e posizionamento di scaffali dell'ambiente di lavoro;
+- realizzazione mini PoC relativo alla lettura e utilizzo di file SVG e comunicazione con Database;
+- incontro con Proponente in data 06-12-23;
+- incontro con professor Cardin in data 07-12-23;
+- redazione del glossario;
+- aggiornamento documento Norme di Progetto;
+- aggiornamento documento Piano di Progetto.
+
+=== Obiettivi mancati
+- miglioramento GitHub Actions.
+
+=== Analisi problematiche insorte
+Durante il meeting di retrospettiva sono emerse le seguenti problematiche da migliorare:
+- mancanza di precise convenzioni da adottare riguardanti il codice;
+- lavoro concentrato principalmente nel weekend;
+- richiesta di maggiore partecipazione dei membri del gruppo sulle board di Miro;
+
+=== Risoluzioni attuate
+Conseguentemente ai problemi rilevati, sono state individuate le relative soluzioni da adottare:
+- formulare convenzioni più chiare da adottare per il codice;
+- impegnarsi a fissare delle scadenze infrasettimanali così da ridurre il carico di lavoro il weekend;
+- utilizzare le board su Miro con anticipo aggiungendo il proprio feedback.
+
+=== Analisi costi di sprint effettivi
+#set text(
+  size: 9pt
+)
+
+#table(
```

Comment:
dato che il preventivo dello sprint 5 era di 545 (seppur non presenta l'assegnazione di programmatori e deve essere corretto dato che avevamo poc in produzione già alla fine dello sprint 4), si può pensare di portare a 2 le ore di programmatore per tutti i programmatori (anzichè 1).

tenendo conto che in settimana sono stati creati e presentati 3 poc, anche eccedere il costo preventivato è comprensibile